runtimes/instance-rootfs/create-debian-12-disk.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -euf
+
+# Variables
+ROOTFS_FILE="./rootfs.btrfs"
+MOUNT_ORIGIN_DIR="/mnt/debian"
+MOUNT_DIR="/mnt/vm"
+IMAGE_URL="https://cloud.debian.org/images/cloud/bookworm/latest/debian-12-genericcloud-amd64.tar.xz"
+IMAGE_NAME="debian-12-genericcloud.tar.xz"
+IMAGE_RAW_NAME="disk.raw"
+
+# Cleanup previous run
+umount "$MOUNT_ORIGIN_DIR" || true
+umount "$MOUNT_DIR" || true
+rm -f "$ROOTFS_FILE"
+
+# Prepare directories
+mkdir -p "$MOUNT_ORIGIN_DIR"
+mkdir -p "$MOUNT_DIR"
+
+# Download Debian image
+echo "Downloading Debian 12 image"
+curl -L "$IMAGE_URL" -o "$IMAGE_NAME"
+
+# Create rootfs.btrfs file
+echo "Creating rootfs.btrfs file"
+dd if=/dev/zero of="$ROOTFS_FILE" bs=1G count=2
```

Comment:
We should avoid using `dd` in our scripts when possible: https://eklitzke.org/the-cult-of-dd

```suggestion
# Allocate a 1GB file
fallocate -l 1G "$ROOTFS_FILE"
```

Reply:
Why 1GB ? Is there a way to shrink it to the minimum required ?